src/main/scala/it/pagopa/interop/attributeregistryprocess/service/impl/PartyRegistryServiceImpl.scala

```diff
@@ -23,25 +23,16 @@ final case class PartyRegistryServiceImpl(
   override def getCategories(bearerToken: String)(implicit contexts: Seq[(String, String)]): Future[Categories] =
     withHeaders { (bearerToken, correlationId, ip) =>
       val request: ApiRequest[Categories] =
-        categoryApi.getCategories(
-          origin = None,
-          page = Some(1),
-          limit = Some(100),
-          xCorrelationId = correlationId,
-          xForwardedFor = ip
-        )(BearerToken(bearerToken))
+        categoryApi.getCategories(origin = None, xCorrelationId = correlationId, xForwardedFor = ip)(
```

Comment:
⚠️ if you do not pass `page` and `limit` parameters, defaults apply and you will receive only the first 10 results.
Here you probably need to loop to retrieve all results, as for other paginations